mms_msg/sampling/utils/distribution_model.py

```diff
@@ -239,16 +265,83 @@ def plot(self, show = False, fig = None, ax = None):
             plt.show()
         return fig, ax
 
+    @staticmethod
+    def to_dict(obj: DistributionModel) -> dict:
+        """
+        Static method that converts a Distribution model into a dictionary.
+
+        Args:
+            obj: DistributionModel which should be converted to a dictionary
+
+        Returns: Dictionary which contains the data of the given object
+        """
+        return obj.__dict__
+
+    @staticmethod
+    def from_dict(dct: dict) -> DistributionModel:
```

Comment:
This seems too complicated. We have `pt.Configurable` for constructing objects from configuration dicts. Is it really necessary that the class is configurable?

I'm pretty sure that `__dict__` does not contain `__module__` and `__class__`, so does `DistributionModel.from_dict(m.to_dict())` work?